src/index.js

```diff
@@ -223,7 +223,18 @@ function querySelectorAll (selector, context = document) {
   return query(selector, context, true)
 }
 
+function getElementsByClassName (classNames, context = document) {
+  const oneSpaceSeparator = classNames.trim().replace(/\s\s+/g, ' ');
```

Comment:
Why `/\s\s+/g` and not `/\s+/g`? We need to replace a single instance of `\t` or `\n` with a space, correct?

Reply:
When replacing with a `'  '` I think the end result is the same. `\s\s+` finds any non-single spaces where `\s+` would return single spaces.

Reply:
Right, but wouldn't it fail with these strings?

```js
"foo\tbar"
"foo\nbar"
```

The single `\t` or `\n` would never get replaced, so the next `.split(' ')` would not work.

Reply:
You're right, thanks! Not sure where I was going with that.